### PR TITLE
add init-job cronJob

### PR DESCRIPTION
**Pull request by bissenbay**
Fixes: https://github.com/thoth-station/init-job/issues/146

### CODE REVIEW

File: openshift/cronJob-template.yaml [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+      schedule: "0 */12 * * *"
+      jobTemplate:
+        spec:
+          template:
+            metadata:
+              labels:
+                app: thoth
+                component: init-job
+            spec:
+              restartPolicy: Never
```
Comment (bissenbay) [inline openshift/cronJob-template.yaml:67]: Change [from](https://github.com/thoth-station/init-job/blob/master/openshift/job-template.yaml#L37) to `Never`

File: openshift/cronJob-template.yaml [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    kind: CronJob
+    metadata:
+      annotations:
+        thoth-station.ninja/template-version: 0.1.0
+      name: init-job
+      labels:
+        app: thoth
+        component: init-job
+    spec:
+      schedule: "0 */12 * * *"
```
Comment (bissenbay) [inline openshift/cronJob-template.yaml:58]: Just randomly chose

File: openshift/cronJob-template.yaml [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+                    - name: THOTH_LOG_INIT_JOB
+                      value: "${THOTH_LOG_INIT_JOB}"
+                    - name: KUBERNETES_API_URL
+                      value: https://kubernetes.default.svc.cluster.local
+                    - name: KUBERNETES_VERIFY_TLS
+                      value: "0"
+                    - name: THOTH_RESULT_API_URL
+                      value: http://result-api
+                    - name: THOTH_INIT_JOB_REGISTER_INDEXES_ONLY
+                      value: "true"
```
Comment (bissenbay) [inline openshift/cronJob-template.yaml:89]: The main different [from](https://github.com/thoth-station/init-job/blob/master/openshift/job-template.yaml), I added this parameter `THOTH_INIT_JOB_REGISTER_INDEXES_ONLY` and set it to `"true"`

File: openshift/cronJob-template.yaml [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+                        secretKeyRef:
+                          name: postgresql
+                          key: database-name
+                  resources:
+                    limits:
+                      memory: '256Mi'
+                      cpu: '500m'
+                    requests:
+                      memory: '256Mi'
+                      cpu: '500m'
```
Comment (bissenbay) [inline openshift/cronJob-template.yaml:140]: There was `test: false` parameter in [job-template](https://github.com/thoth-station/init-job/blob/master/openshift/job-template.yaml#L109) which I deleted here in `cronJob-template`

File: openshift/cronJob-template.yaml (hunk 2 above)
Comment (fridex) [reply openshift/cronJob-template.yaml:58]: Sounds good. :+1:

File: openshift/cronJob-template.yaml [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+                    periodSeconds: 10
+                  env:
+                    - name: THOTH_LOG_INIT_JOB
+                      value: "${THOTH_LOG_INIT_JOB}"
+                    - name: KUBERNETES_API_URL
+                      value: https://kubernetes.default.svc.cluster.local
+                    - name: KUBERNETES_VERIFY_TLS
+                      value: "0"
+                    - name: THOTH_RESULT_API_URL
+                      value: http://result-api
```
Comment (fridex) [inline openshift/cronJob-template.yaml:87]: This one should not be required.

File: openshift/cronJob-template.yaml (hunk 3 above)
Comment (fridex) [reply openshift/cronJob-template.yaml:89]: Please use numerical value - to enable it set it to `"1"`.

File: openshift/cronJob-template.yaml (hunk 5 above)
Comment (bissenbay) [reply openshift/cronJob-template.yaml]: Done

File: openshift/cronJob-template.yaml (hunk 3 above)
Comment (bissenbay) [reply openshift/cronJob-template.yaml]: Thanks, changed.

**Pull request merged**